src/main/java/org/logstash/beats/BeatsHandler.java

```diff
@@ -54,8 +56,14 @@ public void channelRead0(ChannelHandlerContext ctx, Batch batch) throws Exceptio
 
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
+        InetSocketAddress remoteAddress = (InetSocketAddress) ctx.channel().remoteAddress();
+
+        if (remoteAddress != null) {
+            logger.error("Exception: " + cause.getMessage() + ", from: " + remoteAddress.toString());
```

Comment:
Are all exceptions requiring operator action? If so, ERROR log level makes sense. However, I have a suspicion that most network problems will not require operator action to resolve, so maybe this should be `logger.warn` or `logger.info` ?

My thought is that the network is assumed to be unstable, so temporary problems will result in exceptions but otherwise resolve without human intervention, and these temporary problems should not be logged as error or warn.